jackson-modules/jackson-custom-conversions/pom.xml

```diff
@@ -23,6 +23,13 @@
             <artifactId>jackson-datatype-joda</artifactId>
             <version>${jackson.version}</version>
         </dependency>
+        <dependency>
```

Comment:
the other jackson libraries here are using the inherited jackson.version; does this one need a different version?